cobra/preprocessing/preprocessor.py

```diff
@@ -223,28 +223,87 @@ def from_pipeline(cls, pipeline: dict):
             target_encoder,
             is_fitted=pipeline["_is_fitted"],
         )
+    
+    def get_continous_and_discreate_columns(
+        self, 
+        df : pd.DataFrame, 
+        id_col_name : str, 
+        target_column_name :str
+        ) -> tuple:
+        """Filters out the continious and discreate varaibles out of a dataframe and returns a tuple containing lists of column names 
+        It assumes that numerical comumns with less than or equal to 10 different values are categorical
```

Comment:
continuous instead of continious
discrete instead of discrete
variables instead of varaibles
columns instead of columns